rbac/README.md

```diff
@@ -0,0 +1,86 @@
+## RBAC resources for Yugabyte Platform
+
+This directory contains different RBAC manifests containing Roles,
+RoleBindings etc required for the Yugabyte Platform.
+
+### 0. yugabyte-platform-universe-management-sa.yaml (required)
+This is the ServiceAccount whose secret can be used to generate a
+kubeconfig. This should be created in a namespace where your are going
+to install the platform software. It can be created in any other
+namespace as well, it should not be deleted once it is being used by
+the platform.
+
+Make sure you replace the `[yb-platform]` from the following command
+with correct namespace for the ServiceAccount.
+
+```console
+kubectl create ns [yb-platform]
+kubectl apply -n [yb-platform] -f yugabyte-platform-universe-management-sa.yaml
+```
+
+Once the ServiceAccount is created, you can follow any one of the
+following steps depending to your requirements.
+
+### a. platform-global.yaml
+Grants access to manage YugabyteDB universes across all the namespaces
```

Comment:
Grants access to only the specific cluster roles to create and manage YugabyteDB